manifests/s/Sejer/BiblioManuels/1.0.0.0/Sejer.BiblioManuels.installer.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Sejer.BiblioManuels
+PackageVersion: 1.0.0.0

```

Comment:
```suggestion
PackageVersion: 3.5.0
```

Reply:
You'll also need to rename the folder `1.0.0.0` to `3.5.0` to make sure that it matches the version shown in the Control Panel.